### ISSUE TITLE
Compilation error due to "pip install" and missing download pretrained models link

### ISSUE THREAD

**Issue by alessandromarchei** [uzh-rpg/rampvo#11]
Hello,
I would like to use the model to do further work, however i am encountering a couple of problems.

1 during the CUDA kernel compilation, performed during "pip install ."
I have followed the intructions written in the repo, so I have:
-  torch version : 2.6.0+cu124
- gcc and g++ up to 13.2 (I tried every previous versions and got the same result)

While executing the install command, I get this error:
- /home/amarchei/Desktop/repos/rampvo/ramp/altcorr/correlation_kernel.cu(325): error: no suitable conversion function from "const at::DeprecatedTypeProperties" to "c10::ScalarType" exists
- /home/amarchei/Desktop/repos/rampvo/ramp/altcorr/correlation_kernel.cu(299): error: no suitable conversion function from "const at::DeprecatedTypeProperties" to "c10::ScalarType" exists
- /home/amarchei/Desktop/repos/rampvo/ramp/altcorr/correlation_kernel.cu(273): error: no suitable conversion function from "const at::DeprecatedTypeProperties" to "c10::ScalarType" exists
- /home/amarchei/Desktop/repos/rampvo/ramp/altcorr/correlation_kernel.cu(211): error: no suitable conversion function from "const at::DeprecatedTypeProperties" to "c10::ScalarType" exists

They are all relative to the same type of code line, similar to this 
  AT_DISPATCH_FLOATING_TYPES_AND_HALF(net.type(), "patchify_backward_kernel", ([&] {
    patchify_backward_kernel<scalar_t><<<BLOCKS(B * M * D * D), THREADS>>>(radius,
      gradient.packed_accessor32<scalar_t,5,torch::RestrictPtrTraits>(),
      coords.packed_accessor32<float,3,torch::RestrictPtrTraits>(),
      net_gradient.packed_accessor32<scalar_t,4,torch::RestrictPtrTraits>());
  }));

What versions of CUDA, pytorch and g++ have you used?

2 when trying to download the pretrained models at "https://download.ifi.uzh.ch/rpg/web/data/iros24_rampvo/checkpoints/RAMPVO_SingleScale.pth", the page is loaded for a lot of time without returning any result. Are you facing the same issue?

Thank you in advance

**Comment by senecobis** [comment]
Hi, we are facing a temporary issue with our server, that is why data is not downloadable.

For what regards CUDA, the highest version that the installed driver support is CUDA 12.2 as written in the README.

gcc version is 9.4.0.

**Comment by alessandromarchei** [comment]
Thank you for the reply!
So CUDA 12.4 won't work?

**Comment by senecobis** [comment]
No, the highest CUDA that can work is 12.2.
You can change the cuda-toolkit version locally on you conda env `conda install -y nvidia::cuda-toolkit`
